app/helpers/application_helper/toolbar/generic_object_definition_node_center.rb

```diff
@@ -0,0 +1,42 @@
+class ApplicationHelper::Toolbar::GenericObjectDefinitionNodeCenter < ApplicationHelper::Toolbar::Basic
+  button_group('generic_object_definition', [
+    select(
+      :generic_object_definition_configuration,
+      'fa fa-cog fa-lg',
+      title = N_('Configuration'),
+      title,
+      :items => [
+        button(
+          :generic_object_definition_edit,
+          'pficon pficon-edit fa-lg',
+          t = N_('Edit this Generic Object Class'),
+          t,
```

Comment:
The `t = ...` and `t,` are no longer needed.

If you send in just the first string the value of it is used for the other.
See here: https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper/toolbar/basic.rb#L39

Reply:
Addressed in the last commit - 11fc82a

Which is kept as a separate commit for an easier review. 
I can squash it later with the earlier commit, once you are satisfied with the changes.

Reply:
No need fo that. Thx.